hub/apps/how-tos/uno-multiplatform.md

```diff
@@ -0,0 +1,142 @@
+
+:::image type="content" source="images/hello-world/vsix-new-project-options.png" alt-text="Uno solution template for project startup type":::
+
+Click the **Create** button. Wait for the projects to be created and their dependencies to be restored.
+
+A banner at the top of the editor may ask to reload projects, click **Reload projects**:
+:::image type="content" source="images/hello-world/vs2022-project-reload.png" alt-text="Visual Studio banner offering to reload your projects to complete changes":::
+
+## Building your app
+
+Now that you've generated the functional starting point of your multi-platform WinUI application, you can copy markup into it from the Hello World WinUI 3 project outlined in the [previous](/hub/apps/how-tos/hello-world-winui3.md) tutorial.
+
+You should see the following default file structure in your Solution Explorer:
+
+:::image type="content" source="images/hello-world/uno-file-structure.png" alt-text="Default file structure":::
+
+Make sure Visual Studio has your WinUI 3 project open, then copy the child XAML elements from `MainWindow.xaml` in the WinUI 3 project to your `MainPage.xaml` file in the Uno Platform project. The `MainPage` view XAML should look like this:
+
+```xml
+<Page x:Class="HelloWorld.MainPage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:HelloWorld"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  mc:Ignorable="d"
+	  Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+      <!-- Below is the code you copied from MainWindow: -->
+      <StackPanel Orientation="Horizontal"
+                  HorizontalAlignment="Center"
+                  VerticalAlignment="Center">
+          <TextBlock x:Name="myText" 
+                     Text="Hello world!"
+                     Foreground="Red"/>
+      </StackPanel>
+</Page>
+```
+
+* To run the **WebAssembly** (Wasm) head:
+    - Right-click on the `HelloWorld.Wasm` project, select **Set as startup project**
+    - Press the `HelloWorld.Wasm` button to deploy the app
+    - If desired, you can use the `HelloWorld.Server` project as an alternative
+* To debug for **iOS**:
+    - Right-click on the `HelloWorld.Mobile` project, select **Set as startup project**
+    - In the "Debug toolbar" drop-down, select an active iOS device (or the simulator if paired with a Mac):
+
+      :::image type="content" source="images/hello-world/net7-ios-debug.png" alt-text="Visual Studio dropdown to select a target framework to deploy":::
+
+    - Select an active device
+* To debug the **Android** platform:
+    - Right-click on the `HelloWorld.Mobile` project, select **Set as startup project**
+    - In the **Debug toolbar** drop-down, select either an active Android device or the emulator
+        - Select an active device in the "Device" sub-menu
+
```

Comment:
In blank template they'll also get GTK, so you should run that and point out that this can be used on linux